### DISCUSSION TITLE
Best Practice Question

### DISCUSSION THREAD

**Discussion by RonGarlit** [Dotnet-Boxed/Templates#127] [Q&A]
Question.  How come the MVC boilerplate is not using MVC Areas out of the box?  

Since your driving best practices I figured that would be one of the first items you would have set up.  

Thoughts?

Thanks
Ron

**Comment by RehanSaeed** [comment]
Areas are only used by larger sites. We are trying to cater to all. That said, its pretty easy to add areas and this could be an optional feature.

**Comment by RehanSaeed** [comment]
I've been thinking about this one. Areas need a name. I would have to know the name before I could create one which is not possible right now. This could be done by providing the user a text box. This is something I've been thinking about doing for some time but there are a lot of other higher priority features to implement.
